upgrade/README.md

```diff
@@ -91,3 +91,7 @@ CSM 1.6 patch upgrades. Instead, consider upgrading to the latest CSM 1.6 patch
 
 There are no CSM 1.6 patch versions currently available. When any become available, they will
 be listed here.
+
+## Upgrading management and managed nodes with different CSM versions using IUF
```

Comment:
Add this title to the list of headers at the top of the page.

Reply:
Done